references/cli/adopt.cue

```diff
@@ -0,0 +1,179 @@
+import "list"
```

Comment:
let's add a folder code `adopt_plan/` for it.

This one should be named by `native_plan.cue`

Reply:
`native_plan` is a bit ambiguous, since we already have `native` type adoption. `adopt_plan` is not aligned with the arg `--adopt-template`.

I recommend to setup a folder called `adopt-templates` and `default.cue`.

Reply:
`adopt-templates` is fine